docs/csharp/language-reference/statements/jump-statements.md

```diff
@@ -0,0 +1,114 @@
+
+## The `break` statement
+
+The `break` statement terminates the closest enclosing [iteration statement](iteration-statements.md) (that is, `for`, `foreach`, `while`, or `do` loop) or [`switch` statement](selection-statements.md#the-switch-statement). The `break` statement transfers control to the statement that follows the terminated statement, if any.
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/BreakStatement.cs" id="BasicExample":::
+
+In nested loops, the `break` statement terminates only the innermost loop that contains it, as the following example shows:
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/BreakStatement.cs" id="NestedLoop":::
+
+When you use the `switch` statement inside a loop, a `break` statement at the end of a switch section transfers control only out of the `switch` statement. The loop that contains the `switch` statement is unaffected, as the following example shows:
+
+:::code language="csharp" source="snippets/jump-statements/BreakStatement.cs" id="SwitchInsideLoop":::
+
+## The `continue` statement
+
+The `continue` statement starts a new iteration of the closest enclosing [iteration statement](iteration-statements.md) (that is, `for`, `foreach`, `while`, or `do` loop), as the following example shows:
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/ContinueStatement.cs" id="BasicExample":::
+
+## The `return` statement
+
+The `return` statement terminates execution of the function in which it appears and returns control and the function's result, if any, to the caller. The `return` statement can be any of the following two forms:
+
+- Without expression, as in the following example:
+
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/jump-statements/ReturnStatement.cs" id="WithoutExpression":::
+
+  You can use the `return` statement without expression in a function that doesn't compute a value, that is:
+
+  - a [method](../../programming-guide/classes-and-structs/methods.md), [local function](../../programming-guide/classes-and-structs/local-functions.md), or [lambda expression](../operators/lambda-expressions.md) with the result type [`void`](../builtin-types/void.md)
+  - an [async](../keywords/async.md) method, local function, or lambda expression with the result type <xref:System.Threading.Tasks.Task> or <xref:System.Threading.Tasks.ValueTask>
+  - the `set` accessor of a [property](../../programming-guide/classes-and-structs/properties.md) or an [indexer](../../programming-guide/indexers/index.md)
+  - a [constructor](../../programming-guide/classes-and-structs/constructors.md)
+  - a [finalizer](../../programming-guide/classes-and-structs/finalizers.md)
+  - the `add` and `remove` accessors of an [event](../../programming-guide/events/index.md)
```

Comment:
This list is very spec-like. Alternatives:
(1) Remove it at all, leave only the introduction sentence
(2) Shorten it, make it not exhaustive, that is, mention only method and properties as examples
(3) Do nothing, leave as it is
(4) Something else

@BillWagner what do you think?

Reply:
I think it will be easier to read using the introductory sentence structure. Consider something like my suggestion on line 51.